### DISCUSSION TITLE
DPLR: setting of parameter sel_type

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/deepmd-kit#2791] [Q&A]
I am trying to use the DPLR model in the ionic solution, which contains H, B, C, F and N. In the beginning, DW model is trained first with parameter `type`:`dipole`. Since there are many element in the system, I would like to know how to specify the list of parameter `sel_type`, and how to specify which is which, e.g.  bond type 1 between atom F and N, bond type 2 between H and N, etc.

**Comment by wanghan-iapcm** [comment]
https://docs.deepmodeling.com/projects/deepmd/en/master/model/sel.html
no bond needs to be specified between real atoms.

**Reply by ghost** [reply]
Thank you for your reply.

I see. So, in such a case, if the `sel_type` is left unset, there should be 10 types of interaction (according to 2-combination of 5 elemens), and I have to set the list of `model_charge_map` as corresponding to the same number of `sel_type`. 

Is that true?
